test/monitor.js

```diff
@@ -1018,7 +1027,7 @@ describe('Monitor', () => {
 
                         expect(res.statusCode).to.equal(200);
                         // account for hapi15 differences
-                        expect(out.data.length).to.be.greaterThan(0);
+                        expect(out.data.length).to.be.greaterThan(1);
```

Comment:
Drat. I had changed it from 0 to 1, when I was originally adjusting the unit test for wreck to confirm both cases (before I split it into two tests). I will revert that change in the next hour. Thanks!